apps/client/src/app/pages/db/fishing-spot/fishing-spot-bait-datagrid/fishing-spot-bait-datagrid.component.ts

```diff
@@ -0,0 +1,45 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from '@angular/core';
+import { LazyDataService } from 'apps/client/src/app/core/data/lazy-data.service';
+import { combineLatest } from 'rxjs';
+import { filter, map } from 'rxjs/operators';
+import { FishContextService } from '../../service/fish-context.service';
+
+@Component({
+  selector: 'app-fishing-spot-bait-datagrid',
+  template: `<nz-card
+    [nzTitle]="'DB.FISHING_SPOT.Fish_per_bait' | translate"
+    class="expanded"
+    [nzLoading]="loading$ | async"
+    [class.card-with-list]="!(loading$ | async)"
+  >
+    <ng-template #iconRender let-id>
+      <app-item-icon [itemId]="id" width="32"></app-item-icon>
+    </ng-template>
+    <app-fishing-spot-datagrid
+      [activeFish]="activeFish"
+      (activeFishChange)="activeFishChange.next($event)"
+      [rowIconRender]="iconRender"
+      [table]="table$ | async"
+    ></app-fishing-spot-datagrid>
+  </nz-card>`,
```

Comment:
Is there a specific reason for this template to be inline rather than inside its own file?

Reply:
Nope! I'll break it out into it's own file.